codalab/apps/web/tasks.py

```diff
@@ -816,9 +816,19 @@ def send_mass_email(competition_pk, body=None, subject=None, from_email=None, to
     text = render_to_string("emails/notifications/participation_organizer_direct_email.txt", context)
     html = render_to_string("emails/notifications/participation_organizer_direct_email.html", context)
 
-    mail_tuples = ((subject, text, html, from_email, [e]) for e in to_emails)
+    message_info = []
+    site = Site.objects.get_current()
 
-    _send_mass_html_mail(mail_tuples)
```

Comment:
ah, does this also fix a user context problem?

By the way, we recently had a problem where emailing participants count > 100 seems to not be sending for some reason.. if you setup a free SendGrid account you can send a couple hundred emails to see if you can replicate this recent problem?

Reply:
Or: maybe something was branched wrong?

Reply:
I believe this was a branching problem. I purged my local workspace of old commits, so this problem does not keep appearing.